conans/migrations.py

```diff
@@ -39,3 +41,26 @@ def _load_old_version(self):
         except Exception:
             old_version = None
         return old_version
+
+    @staticmethod
+    def can_be_overwritten(file_path):
```

Comment:
Do not make it an independent method, embed it in the ClientMigrator ``_update_file``